src/clr-angular/utils/focus/key-focus/roving-tabindex.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Component, ElementRef, Renderer2, Input } from '@angular/core';
+
+import { KeyCodes } from '../../../utils/enums/key-codes.enum';
+import { ClrKeyFocus } from './key-focus';
+import { FocusableItem } from './interfaces';
+
+@Component({
+  selector: '[clrRovingTabindex]',
+  template: '<ng-content></ng-content>',
+})
+export class ClrRovingTabindex extends ClrKeyFocus {
```

Comment:
We removed all this code from the ClrKeyFocus a week before I needed it here. Without it, I'd have to spread tabindex calculation code all throughout the hierarchy of combobox inner components. Now it takes 5 lines of code overall.
Shijir needed KeyFocus without the tabindex. So, we agreed his case is valid and the code was removed. But the Roving Tabindex is a recognized ARIA pattern that we may need on many places and it steps upon the ClrKeyFocus. 
Alternatively, we can just duplicate the code from ClrKeyFocus here. I don't see advantages to it though.

Reply:
As we discussed, there are other approaches but at this point I don't know that we want to make any changes. We just want to have this in mind and long term this might get sorted out.